packages/experimental/Avatar/src/JSAvatarTokens.windows.ts

```diff
@@ -0,0 +1,271 @@
+    },
+    darkRed: {
+      ...getColorProps('darkRed', t),
+    },
+    cranberry: {
+      ...getColorProps('cranberry', t),
+    },
+    red: {
+      ...getColorProps('red', t),
+    },
+    pumpkin: {
+      ...getColorProps('pumpkin', t),
+    },
+    peach: {
+      ...getColorProps('peach', t),
+    },
+    marigold: {
+      ...getColorProps('marigold', t),
+    },
+    gold: {
+      ...getColorProps('gold', t),
+    },
+    brass: {
+      ...getColorProps('brass', t),
+    },
+    brown: {
+      ...getColorProps('brown', t),
+    },
+    forest: {
+      ...getColorProps('forest', t),
+    },
+    seafoam: {
+      ...getColorProps('seafoam', t),
+    },
+    darkGreen: {
+      ...getColorProps('darkGreen', t),
+    },
+    lightTeal: {
+      ...getColorProps('lightTeal', t),
+    },
+    teal: {
+      ...getColorProps('teal', t),
+    },
+    steel: {
+      ...getColorProps('steel', t),
+    },
+    blue: {
+      ...getColorProps('blue', t),
+    },
+    royalBlue: {
+      ...getColorProps('royalBlue', t),
+    },
+    cornflower: {
+      ...getColorProps('cornflower', t),
+    },
+    navy: {
+      ...getColorProps('navy', t),
+    },
+    lavender: {
+      ...getColorProps('lavender', t),
+    },
+    purple: {
+      ...getColorProps('purple', t),
+    },
+    grape: {
+      ...getColorProps('grape', t),
+    },
+    lilac: {
+      ...getColorProps('lilac', t),
+    },
+    pink: {
+      ...getColorProps('pink', t),
+    },
+    magenta: {
+      ...getColorProps('magenta', t),
+    },
+    plum: {
+      ...getColorProps('plum', t),
+    },
+    beige: {
+      ...getColorProps('beige', t),
+    },
+    mink: {
+      ...getColorProps('mink', t),
+    },
+    platinum: {
+      ...getColorProps('platinum', t),
+    },
+    anchor: {
+      ...getColorProps('anchor', t),
+    },
+  } as JSAvatarTokens);
+
+/**
+ * A function which returns object of props depending on color and theme.
+ * @param color
+ * @param theme
+ * @returns object of props - backgroundColor, color and ringColor
+ */
+function getColorProps(color: string, theme: Theme) {
+  const themeAppearance = theme.host.appearance;
+  switch (themeAppearance) {
+    case 'light':
+    default:
+      return {
+        backgroundColor: globalTokens.color[color].tint40,
+        color: globalTokens.color[color].shade30,
+        iconColor: globalTokens.color[color].shade30,
+        ringColor: globalTokens.color[color].primary,
+      };
+    case 'dark':
+      return {
+        backgroundColor: globalTokens.color[color].shade30,
+        color: globalTokens.color[color].tint40,
+        iconColor: globalTokens.color[color].tint40,
+        ringColor: globalTokens.color[color].tint30,
+      };
+    case 'highContrast':
+      return {
+        backgroundColor: globalTokens.color.black,
+        color: globalTokens.color.white,
+        iconColor: globalTokens.color.white,
+        ringColor: theme.colors.brandBackgroundSelected,
```

Comment:
These entries should not use global tokens, because they will not adjust to user HC theme settings on the system. There should be some aliasTokens we should be using here instead. Borrowing the values for the neutral colors may suffice here.

Reply:
fixed